source/agora/consensus/protocol/Nominator.d

```diff
@@ -916,6 +916,7 @@ extern(D):
             if (fail_reason == this.ledger.InvalidConsensusDataReason.NotInPool)
             {
                 log.trace("validateValue(): This node can not yet fully validate this value: {}. Data: {}", fail_reason, data.prettify);
+                this.network.getUnknownTXs(this.ledger);
```

Comment:
Having a validation method trigger a bunch of network requests and yield is a bomb (impatiently) waiting to go off.

Reply:
Updated to set a timer which will start a `Task` which calls the `getUnkownTXs`.